libraries/core-react/src/components/Table/Table.test.tsx

```diff
@@ -41,3 +55,72 @@ describe('Caption', () => {
     )
   })
 })
+
+describe('Table', () => {
+  it('Can render a cell as a header cell', () => {
+    const text = 'Name'
+    const { getByText, container } = render(
+      <Table>
+        <Head>
+          <Row>
+            <Cell as="th">{text}</Cell>
+          </Row>
+        </Head>
+      </Table>,
+    )
+
+    const headerCell = getByText(text)
+    const th = container.querySelector('th')
+    expect(headerCell).toEqual(th)
+  })
+
+  it('Renders the header row visually different than the other rows by using a background colour', () => {
+    const header = 'Header'
+    const body = 'Body content'
+    const { getByText } = render(
+      <Table>
+        <Head>
+          <Row>
+            <Cell as="th">{header}</Cell>
+          </Row>
+        </Head>
+        <Body>
+          <Row>
+            <Cell>{body}</Cell>
+          </Row>
+        </Body>
+      </Table>,
+    )
+    const headElement = getByText(header).closest('thead')
+    const regularContent = getByText(body)
+    expect(headElement).not.toHaveStyleRule('background', 'rgba(255,255,255,1)')
+    expect(regularContent).toHaveStyleRule('background', 'rgba(255,255,255,1)')
+  })
+
+  it('Adjusts font for better readability if the text is a number', () => {
+    const text = '369470'
+    const { getByText } = render(
+      <Table>
+        <Body>
+          <Row>
+            <Cell variant="numeric">{text}</Cell>
+          </Row>
+        </Body>
+      </Table>,
+    )
+
+    expect(getByText(text)).toHaveStyleRule(
+      'font-feature-settings',
+      "'tnum' on,'lnum' on",
```

Comment:
This should be tested against the `DataCell` tokens value for numeric cells

Reply:
Yes, probably... But it gives 
![Skjermbilde 2020-12-14 kl  10 27 59](https://user-images.githubusercontent.com/1568252/102064052-1eed0b80-3df7-11eb-9c6b-9261eb7253a4.png)

Reply:
Tried to fix this in figma-broker, but it didn't work that well... Seems like prettier is re adding the whitespace or something 🤯  Cheated and  trimmed the text. I'm not sure if the world is a better place or not after this change 🤷‍♀️  @mimarz